src/base/index.vue

```diff
@@ -600,6 +600,7 @@ export default {
         this.selectedVariation = this.activeVariation.pageName;
         this.loading = false;
         this.$emit("load", true);
+        this.$cookies.set("sections-page-timestamp", res.data.last_updated);
```

Comment:
do you need to store that in a cookie ? if the page gets refreshed it will load a new version of it... storing it in a cookie is not needed here, it can be kept in a temporary variable that gets reset on page refresh